### PR TITLE
[zh] Modify ambiguous sentences in high-availability.md

### PR DESCRIPTION
**Pull request by gc235**
Modify ambiguous sentences

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle [zh] Modify ambiguous sentences in high-availability.md

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ option. Your cluster requirements may need a different configuration.
    you must add the `certificateKey` field in the appropriate config locations
    (under `InitConfiguration` and `JoinConfiguration: controlPlane`).
    -->
-   标志 `kubeadm init`、`--config` 和 `--certificate-key` 不能混合使用，
+   在使用 kubeadm init 时，标记 --config 和 --certificate-key 不能混用，
```
Comment (tengqm) [inline content/zh-cn/docs/setup/production-environment/tools/kubeadm/high-availability.md:317]:
```suggestion
   在使用 `kubeadm init` 时，标记 `--config` 和 `--certificate-key` 不能混用，
```

Comment (gc235) [reply content/zh-cn/docs/setup/production-environment/tools/kubeadm/high-availability.md]: Thanks

**Review by gc235 — COMMENTED** [review]
Apply suggestions from code review

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Review by windsonsea — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**